packages/icons/svg/click.svg

```diff
@@ -0,0 +1,10 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+<g id="icon/click">
+<mask id="mask0_3898_345110" style="mask-type:alpha" maskUnits="userSpaceOnUse" x="2" y="2" width="20" height="20">
+<path id="Union" fill-rule="evenodd" clip-rule="evenodd" d="M13 3C13 2.44772 12.5523 2 12 2C11.4477 2 11 2.44772 11 3V6C11 6.55228 11.4477 7 12 7C12.5523 7 13 6.55228 13 6V3ZM3 11C2.44772 11 2 11.4477 2 12C2 12.5523 2.44772 13 3 13H6C6.55228 13 7 12.5523 7 12C7 11.4477 6.55228 11 6 11H3ZM4.89287 4.8929C5.28339 4.50238 5.91656 4.50238 6.30708 4.8929L8.50708 7.0929C8.89761 7.48342 8.89761 8.11659 8.50708 8.50711C8.11656 8.89764 7.48339 8.89764 7.09287 8.50711L4.89287 6.30711C4.50234 5.91659 4.50234 5.28342 4.89287 4.8929ZM19.1071 6.30711C19.4976 5.91659 19.4976 5.28342 19.1071 4.8929C18.7166 4.50237 18.0834 4.50238 17.6929 4.8929L15.4929 7.0929C15.1024 7.48342 15.1024 8.11659 15.4929 8.50711C15.8834 8.89764 16.5166 8.89764 16.9071 8.50711L19.1071 6.30711ZM8.50708 15.4929C8.89761 15.8834 8.89761 16.5166 8.50708 16.9071L6.30708 19.1071C5.91656 19.4976 5.28339 19.4976 4.89287 19.1071C4.50234 18.7166 4.50234 18.0834 4.89287 17.6929L7.09287 15.4929C7.48339 15.1024 8.11656 15.1024 8.50708 15.4929ZM12.3162 11.0513C11.9569 10.9315 11.5607 11.0251 11.2929 11.2929C11.0251 11.5607 10.9315 11.9569 11.0513 12.3162L14.0513 21.3162C14.1792 21.6999 14.5257 21.9688 14.9291 21.9975C15.3326 22.0262 15.7135 21.809 15.8944 21.4472L17.7454 17.7454L21.4472 15.8944C21.809 15.7135 22.0262 15.3326 21.9975 14.9291C21.9688 14.5257 21.6999 14.1792 21.3162 14.0513L12.3162 11.0513ZM15.1852 18.3934L13.5811 13.5811L18.3934 15.1852L16.5528 16.1056C16.3593 16.2023 16.2023 16.3593 16.1056 16.5528L15.1852 18.3934Z" fill="#1B1B20"/>
```

Comment:
@jpyzio123 Hey! Please replace the color value inside the `fill` attribute with `currentColor` so React can bind props properly and use dynamic color value instead of hadcoded hash.

Reply:
hey! sorry, first time adding icon 😅 fixed - https://github.com/livechat/design-system/pull/1276/commits/0a52fd583adb1828bd2b1c7fdca81019638858e8